src/pudl/transform/eia.py

```diff
@@ -360,7 +361,6 @@ def _manage_strictness(col, eia860m):
 def harvesting(  # noqa: C901
     entity: str,
```

Comment:
this is a nit but we might want to make this a literal.

Reply:
Yeah, or an Enum even.

Reply:
I've enumerated the entities!